scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl

```diff
@@ -219,6 +226,29 @@ workflow CNVSomaticPairWorkflow {
             gatk_docker = gatk_docker,
             disk_space_gb = plot_normal_disk
     }
+
+    if (is_run_oncotator) {
+        call Oncotate.CNVOncotateCalledSegments as OncotateCalledCNVWorkflow {
+            input:
+                 called_file = CallCopyRatioSegmentsTumor.called_copy_ratio_segments,
+                 oncotator_docker = oncotator_docker
+        }
+    }
+
+    output {
+        File modeled_segments_tumor = ModelSegmentsTumor.modeled_segments
+        File called_copy_ratio_segments_tumor = CallCopyRatioSegmentsTumor.called_copy_ratio_segments
+        File denoised_read_counts_tumor = DenoiseReadCountsTumor.denoised_copy_ratios
+        File standardized_copy_ratios_tumor = DenoiseReadCountsTumor.standardized_copy_ratios
+
+        File modeled_segments_normal = ModelSegmentsNormal.modeled_segments
+        File called_copy_ratio_segments_normal = CallCopyRatioSegmentsNormal.called_copy_ratio_segments
+        File denoised_read_counts_normal = DenoiseReadCountsNormal.denoised_copy_ratios
+        File standardized_copy_ratios_normal = DenoiseReadCountsNormal.standardized_copy_ratios
+
+        File oncotated_called_file_tumor = select_first([OncotateCalledCNVWorkflow.oncotated_called_file, "null"])
```

Comment:
Is there a nicer way to specify optional outputs?  Or is this the conventional way?

Reply:
This was a necessary workaround in previous versions of cromwell.  I can double check that this is no longer needed. 

@jsotobroad Do I still need to do this to handle issues with optional output parameters?  In cromwell 29...

Reply:
No further action.  I spoke to @jsotobroad , we are going to look into this, but any changes will be in a different PR.

Reply:
Never mind, going to test it here.